docs/2.10.0/docs/canton/usermanual/contract_keys.rst

```diff
@@ -309,20 +329,31 @@ Caveats:
 
 - Before creating a contract with the key ``k`` for the first time, your application must create the matching ``KeyState`` contract with ``allocated`` set to ``False``.
   Such a contract must be created at most once.
-  Most likely, you will want to choose a "master" participant on which you create such contracts.
-- Do not delegate choices on the ``Keyed`` contract to parties other than the maintainers.
-- You must never send a command that creates or archives the ``Keyed`` contract directly.
+  Most likely, you will want to choose a "primary" participant on which you create such contracts.
+- ``Keyed`` contracts must never be created or archived directly,
+  as this would break the synchronization with the corresponding ``KeyState`` contract.
```

Comment:
```suggestion
-  With a small number of exceptions, ``Keyed`` contracts must never be created or archived directly;
  this would break the synchronization with the corresponding ``KeyState`` contract.
```